tests/test_arbitrage.py

```diff
@@ -30,16 +31,27 @@ def setUp(self):
 
 class APITest(unittest.TestCase):
     def setUp(self):
-        self.api = arbitrage.API()
-
-    def test_get_option_expirations(self):
-        pass
-
-    def test_get_option_chain(self):
-        pass
-
-    def test_doesnt_exist(self):
-        pass
+        self.api = arbitrage.API("tradier")
+
+    def test_response(self):
+        r = requests.get('https://sandbox.tradier.com/v1/markets/options/expirations',
```

Comment:
what the fuck is this?

Use the class you just made and make this TradierAPITest. Why are you using the raw requests here????

Reply:
fixed